contrib/fwupd.spec.in

```diff
@@ -301,6 +301,7 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %{_libdir}/fwupd-plugins-3/libfu_plugin_rts54hub.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_steelseries.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_superio.so
+%{_libdir}/fwupd-plugins-3/libfu_plugin_synaprom.so
```

Comment:
Do you know that "Prometheus" API is significantly different enough than other Synaptics finger print reader?  If they're relatively the same I'd say synaptics_fingerprint.so instead.

Reply:
I've asked for clarification from Synaptics about this; synaprom was really just a placeholder until we got some clarity on trademark usage and that kind of thing.

Reply:
it's syaptics_prometheus now.